packages/txservice/src/error.ts

```diff
@@ -356,14 +350,23 @@ export const parseError = (error: any): NxtpError => {
     return new TransactionReverted(TransactionReverted.reasons.GasExceedsAllowance, undefined, context);
   } else if (
     message.match(
-      /another transaction with same nonce|same hash was already imported|transaction nonce is too low|nonce too low|already known/,
+      /another transaction with same nonce|same hash was already imported|transaction nonce is too low|nonce too low|oldnonce/,
```

Comment:
should there be a space? have we seen this string? what about `already known` -- why take it out?

Reply:
no space, this is from matic providers I believe

already known is what happens when Fallback Provider spams every provider with a new sent TX
but by the time it spams 1 or more of the providers they are like 'I already seen it dude'

AlreadyKnown is now a sep. error and it's only used internally bc it shouldn't be raised to consumer level.

Reply:
see ServerError.reasons.TransactionAlreadyKnown below